.rubocop.yml

```diff
@@ -32,7 +32,7 @@ Metrics/AbcSize:
   Enabled: false
 PerceivedComplexity:
   Enabled: false
-SingleSpaceBeforeFirstArg:
+Style/SingleSpaceBeforeFirstArg:
```

Comment:
Great catch, I was scratching my head here when it spewed an error a few minutes ago but decided to leave it be.